src/mobile/components/Profile/ApiKeysItem/index.tsx

```diff
@@ -0,0 +1,61 @@
+import classnames from 'classnames';
+import * as React from 'react';
+import { Form } from 'react-bootstrap';
+import { useIntl } from 'react-intl';
+import { localeDate } from '../../../../helpers';
+
+const ApiKeysItemComponent = props => {
+    const { index, item } = props;
+    const intl = useIntl();
+
+    const createdAt = localeDate(item.created_at, 'fullDate');
+    const updatedAt = localeDate(item.updated_at, 'fullDate');
```

Comment:
we can wrap it in React.useMemo

`const createdAt = React.useMemo(() => localeDate(item.created_at, 'fullDate'), [item.created_at])`

Reply:
okay, I'll change it in the next patch